sqlglot/generator.py

```diff
@@ -2111,17 +2123,7 @@ def offset_limit_modifiers(
         ]
 
     def after_having_modifiers(self, expression: exp.Expression) -> t.List[str]:
```

Comment:
i think we don't need this method anymore which is nice

Reply:
we can improve now select, and remove other things like

prewhere, laterals, connect, match etc

Reply:
Can take a look. Btw, one detail with these dicts is that we need to make sure we remember to define the kv pairs in the right order. For example, I intentionally put `"sort"` after `"distribute"` here, because that's the correct order.

Reply:
just add a test

Reply:
Cool we should have a few tests already